src/main/java/org/broadinstitute/hellbender/tools/longreads/graph/AlignedBaseVertex.java

```diff
@@ -0,0 +1,113 @@
+package org.broadinstitute.hellbender.tools.longreads.graph;
+
+import org.broadinstitute.hellbender.tools.walkers.haplotypecaller.graphs.SeqVertex;
+
+import java.util.Objects;
+
+public class AlignedBaseVertex extends SeqVertex implements Comparable<AlignedBaseVertex>  {
+    private String readName;
+    private GenomicAndInsertionPosition pos;
+
+    public AlignedBaseVertex(final byte[] bases,
+                             final String contig,
+                             final int genomePos,
+                             final int insertionOffset,
+                             final String readName) {
+        super(bases);
+        this.pos = new GenomicAndInsertionPosition(contig, genomePos, insertionOffset);
+        this.readName = readName;
+    }
+
+    public AlignedBaseVertex(final byte[] bases,
+                             final GenomicAndInsertionPosition pos,
+                             final String readName) {
+        super(bases);
+        this.pos = pos;
+        this.readName = readName;
+    }
+
+
+    public String getReadName() { return readName; }
+    public GenomicAndInsertionPosition getPos() { return pos; }
+    public boolean isSingleBase() { return getSequence().length == 1; }
+
+    /**
+     * Check to see if the given sequence and the sequence of {@code this} {@link AlignedBaseVertex} are the same.
+     * @param seq The {@code byte[]} containing a sequence to check for equality to the sequence in {@code this} {@link AlignedBaseVertex}.
+     * @return {@code true} iff the given sequence and the sequence of {@code this} {@link AlignedBaseVertex} are the same.
+     */
+    public boolean isSequenceEqual(final byte[] seq ) {
+        if ( getSequence().length == seq.length ) {
```

Comment:
There's got to be some sort of array comparison method for this, right?

Reply:
`Arrays.equals`.  Fixed.